doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -372,13 +356,9 @@ temperature<=20
   matches entities which don't have a type.</li>
 </ul>
 
-</td></tr>
-<tr><td style="padding-right: 40px; width: 140px;">geometry</td><td>
-
-<p>Defines a geografical area so only the entities located in that area matches the query. It is composed of a tokens list
-separated by <code>;</code>. The first token is the shape of the geometry, the rest of the tokens (if any) depends
-on the shape. In particular, the following geometries can be defined so far:</p>
-
+<p><code>geometry</code> Defines a geografical area so only the entities located in that area matches the query.
```

Comment:
There is no separation between `geometry` and the `q` statement explanation. Thus, the reader may think (wrongly) that `geometry` is a kind of `q` statement. Let's remember that the list entities operations support three diferent parameters: `q` (make of statements), `geometry` and `coords`. I would suggest to describe `q` in a sub-section of the "Simple Query Language" section, then the geometry parameters (`geometry` and `coords`) in other sub-section.